test/test_runner.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var Metamocha = require('../lib/metamocha');
+
+// Instantiate new Metamocha
+var metamocha = new Metamocha();
+
+// Add files from directory
+metamocha.addFolder('test/');
+
+// Apply a filter
+metamocha.addFilter({
```

Comment:
Unless you intend to support multiple keys in the object passed in? Either way, I would support a function OR an object (if you in fact need more keys)